cmake/find_llvm.cmake

```diff
@@ -1,6 +1,6 @@
 # Broken in macos. TODO: update clang, re-test, enable
 if (NOT APPLE)
-    option (ENABLE_EMBEDDED_COMPILER "Set to TRUE to enable support for 'compile' option for query execution" 1)
+    option (ENABLE_EMBEDDED_COMPILER "Set to TRUE to enable support for 'compile' option for query execution" 0)
```

Comment:
Why we need to disable it?

Reply:
Fixed. We don't need to disable it. it was turned off in my test env to avoid compiling llvm lib.